src/pages/Account/Venue/VenueMapEditor/DraggableSubvenue.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { CSSProperties, useEffect } from "react";
+import { DragSourceMonitor, useDrag } from "react-dnd";
+import { getEmptyImage } from "react-dnd-html5-backend";
+import { Resizable } from "re-resizable";
+
+import { Dimensions } from "types/utility";
+
+import { SubVenueIcon } from "./Container";
+import { ItemTypes } from "./ItemTypes";
+
+import styles from "./DraggableSubvenue.module.scss";
+
+const getStyles = (
+  left: number,
+  top: number,
+  isDragging: boolean
+): React.CSSProperties => ({
+  position: "absolute",
+  top,
+  left,
+  opacity: isDragging ? 0 : 1,
+  height: isDragging ? 0 : "",
+  background: "rgba(147, 124, 99, 0.2)",
+});
+
+export type PropsType = SubVenueIcon & {
+  imageStyle?: CSSProperties;
```

Comment:
[NOTE] This type is basically what something named `styles` would be since `import styles from "./some.module.scss"` doesn't import `CSSProperties` but `Record<string,string>` of the mapped class names.